docker/run_jakartaeetck.sh

```diff
@@ -406,88 +409,64 @@ ant config.vi.javadb
-cd $TS_HOME/bin;
+cd "$TS_HOME/bin";
 echo "ant start.auto.deployment.server > /tmp/deploy.out 2>&1 & "
 ant start.auto.deployment.server > /tmp/deploy.out 2>&1 &
 ### ctsStartStandardDeploymentServer.sh ends here #####
 
-cd $TS_HOME/bin;
+printf  "
+******************************************************
+* Executing Ant-based Tests ...                      *
+******************************************************
+
+"
+
+cd "$TS_HOME/bin";
 if [ -z "$KEYWORDS" ]; then
   ant -f xml/impl/glassfish/s1as.xml run.cts -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Dtest.areas="${test_suite}"
 else
   ant -f xml/impl/glassfish/s1as.xml run.cts -Dkeywords=\"${KEYWORDS}\" -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Dtest.areas="${test_suite}"
 fi
 
 
-cd $TS_HOME/bin;
+cd "$TS_HOME/bin";
 # Check if there are any failures in the test. If so, re-run those tests.
 FAILED_COUNT=0
 ERROR_COUNT=0
-TEST_SUITE=`echo "${test_suite}" | tr '/' '_'`
-FAILED_COUNT=`cat ${JT_REPORT_DIR}/${TEST_SUITE}/text/summary.txt | grep 'Failed.' | wc -l`
-ERROR_COUNT=`cat ${JT_REPORT_DIR}/${TEST_SUITE}/text/summary.txt | grep 'Error.' | wc -l`
+FAILED_COUNT=$(cat ${JT_REPORT_DIR}/${TEST_SUITE}/text/summary.txt | grep 'Failed.' | wc -l)
+ERROR_COUNT=$(cat ${JT_REPORT_DIR}/${TEST_SUITE}/text/summary.txt | grep 'Error.' | wc -l)
 if [[ $FAILED_COUNT -gt 0 || $ERROR_COUNT -gt 0 ]]; then
   echo "One or more tests failed. Failure count:$FAILED_COUNT/Error count:$ERROR_COUNT"
   echo "Re-running only the failed, error tests"
-if [ -z "$KEYWORDS" ]; then
-  ant -f xml/impl/glassfish/s1as.xml run.cts -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Drun.client.args="-DpriorStatus=fail,error"  -DbuildJwsJaxws=false -Dtest.areas="${test_suite}"
-else
-  ant -f xml/impl/glassfish/s1as.xml run.cts -Dkeywords=\"${KEYWORDS}\" -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Drun.client.args="-DpriorStatus=fail,error"  -DbuildJwsJaxws=false -Dtest.areas="${test_suite}"
-fi
+  if [ -z "$KEYWORDS" ]; then
+    ant -f xml/impl/glassfish/s1as.xml run.cts -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Drun.client.args="-DpriorStatus=fail,error"  -DbuildJwsJaxws=false -Dtest.areas="${test_suite}"
+  else
+    ant -f xml/impl/glassfish/s1as.xml run.cts -Dkeywords=\"${KEYWORDS}\" -Dant.opts="${CTS_ANT_OPTS} ${ANT_OPTS}" -Drun.client.args="-DpriorStatus=fail,error"  -DbuildJwsJaxws=false -Dtest.areas="${test_suite}"
+  fi
 
   # Generate combined report for both the runs.
-  ant -Dreport.for=com/sun/ts/tests/$test_suite -Dreport.dir=${JT_REPORT_DIR}/${TEST_SUITE} -Dwork.dir=${JT_WORK_DIR}/${TEST_SUITE} report
-fi
-
-export HOST=`hostname -f`
-echo "1 ${TEST_SUITE} ${HOST}" > ${CTS_HOME}/args.txt
-mkdir -p ${WORKSPACE}/results/junitreports/
-${JAVA_HOME}/bin/java -Djunit.embed.sysout=true -jar ${TS_HOME}/docker/JTReportParser/JTReportParser.jar ${CTS_HOME}/args.txt ${JT_REPORT_DIR} ${WORKSPACE}/results/junitreports/
-rm -f ${CTS_HOME}/args.txt
-
-if [ -z ${vehicle} ];then
-  RESULT_FILE_NAME=${TEST_SUITE}-results.tar.gz
```

Comment:
After this change is merged, we should validate that the results are still collected like https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/10.0.x/7/artifact/ejb30_bb-results.tar.gz (looks like they should be).

Reply:
You are right, for some reason it still works in my builds, but not on Eclipse's. I have to fix it.